Sparkle/SUTouchBar.m

```diff
@@ -0,0 +1,94 @@
+//
+//  SUTouchBar.m
+//  Sparkle
+//
+//  Created by Yuxin Wang on 05/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+#import "SUTouchBar.h"
+
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101202
```

Comment:
Last looking at the code, I don't think we need class-from-string trickery because of hooking into the responder chain. The SDK check is more about compiling with older Xcode versions which shouldn't be the highest priority right now (In fact I'd be more interested currently in seeing if Travis can compile with touch bar support after upgrading Xcode 8 image).